src/path/cache.rs

```diff
@@ -23,33 +24,29 @@ bitflags! {
 
 #[derive(Copy, Clone, Debug, Default)]
 pub struct Point {
-    x: f32,
-    y: f32,
-    dx: f32,
-    dy: f32,
+    pos: Position,
+    dpos: Position,
```

Comment:
`dx`/`dy` is a vector, not a position.

Reply:
I guess, I should just rename Position to Vector

Reply:
Positions are not vectors, but a vector can always be used to represent a position.

Reply:
I would greatly prefer having both positions and vectors. Otherwise it is not clear what the data represents.